aws/v9.0.7/release.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v9.0.7
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - name: chart-operator
+    version: 0.13.0
+  - componentVersion: 1.16.5
+    name: cluster-autoscaler
+    version: 1.16.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 1.9.5
+    name: kube-state-metrics
+    version: 1.1.0
+  - componentVersion: 0.3.3
+    name: metrics-server
+    version: 1.1.0
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 0.34.0
+    name: nginx-ingress-controller
+    version: 1.7.2
+  - componentVersion: 0.18.1
+    name: node-exporter
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 5.5.3
+  - name: calico
+    version: 3.9.6
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 0.23.12
+  - name: containerlinux
+    version: 2512.2.1
```

Comment:
```suggestion
    version: 2512.2.0
```

Reverting flatcar upgrade since it doesn't work with existing aws-operator 5.5.3

```
E 07/15 08:44:08 /apis/provider.giantswarm.io/v1alpha1/namespaces/default/awsconfigs/jp27a failed processing event | operatorkit/controller/controller.go:391 | controller=aws-operator | event=update | loop=1617 | version=277085353
	/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/controller/controller.go:579:
	/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/resource/wrapper/metricsresource/resource_wrapper.go:39:
	/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/resource/wrapper/retryresource/resource_wrapper.go:59:
	/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/backoff/retry.go:23:
	/go/src/github.com/giantswarm/aws-operator/vendor/github.com/giantswarm/operatorkit/resource/wrapper/retryresource/resource_wrapper.go:47:
	/go/src/github.com/giantswarm/aws-operator/service/controller/resource/tccp/create.go:88:
	/go/src/github.com/giantswarm/aws-operator/service/controller/resource/tccp/create.go:163:
	/go/src/github.com/giantswarm/aws-operator/service/controller/resource/tccp/create.go:307:
	/go/src/github.com/giantswarm/aws-operator/service/controller/key/key.go:754: no image id for version '2512.2.1'
	not found error
```

//cc @MartaD @MarcelMue @cokiengchiara